extensions/mssql/src/objectManagement/utils.ts

```diff
@@ -43,7 +43,16 @@ export function isValidSQLPassword(password: string, userName: string = 'sa'): b
 	return !containsUserName && password.length >= 8 && password.length <= 128 && (hasUpperCase + hasLowerCase + hasNumbers + hasNonAlphas >= 3);
 }
 
-// Converts number to two decimal placed string
+/*
+* Converts number to two decimal placed string
+*/
 export function convertNumToTwoDecimalStringinMB(value: number): string {
 	return localizedConstants.StringValueInMB(value?.toFixed(2));
 }
+
+/*
+* Converts string value coming from STS to pascal case, (first character to uppercase)
+*/
+export function toPascalCase(value: string): string {
```

Comment:
This looks trivial, but it's also very easy to unit test. Can we add some tests please 😄

Reply:
Sure, working on adding a tests with this PR, will add soon